frontend/components/new/new-droplet-form.tsx

```
@@ -58,10 +64,12 @@ export function CreateDropletForm({
   const [submissionState, setSubmissionState] = useState(
     initialSubmissionState,
   );
+  const [existingDropletSlug, setExistingDropletSlug] = useState<string | null>(null);
```

ESLint

[error] 'existingDropletSlug' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)